### DISCUSSION TITLE
No Tooltips for Icons... // enterprise deployment with config

### DISCUSSION THREAD

**Discussion by jens2k4** [Devolutions/UniGetUI#1564] [General]
On the software update page, there are no tooltips for the icons. (shield, Rhombus etc). maybe only in german language? 

is it possible to deploy wingetUI in the enterprise (via intune) and push the config also to the users?

**Comment by marticliment** [comment]
> On the software update page, there are no tooltips for the icons. (shield, Rhombus etc). maybe only in german language?

This issue has been fixed. Next release will have tooltips



> is it possible to deploy wingetUI in the enterprise (via intune) and push the config also to the users?

Pushing config is not supported yet, but if you copy the .wingetui folder located on your user directory you will be able to effectively load and restore settings to different machines
